build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
```

Comment:
Not sure we need this script in the repo, maybe you can add it to the readme instead. Just a suggestion...